tap_helpshift/__init__.py

```diff
@@ -91,6 +92,17 @@ def __init__(self, client, catalog, config):
         self.catalog = catalog
 
         self.start_date = config['start_date']
+        self.is_secondary_syncer = config.get('is_secondary_syncer', False)
+        # To keep issue_analytics up to date, we have a secondary tap running that syncs
+        # from the beginning of num_days_to_sync ago till now only for issues and issue_analytics.
+        # This window can be fairly large so long as it's during non-busy hours
+        if self.is_secondary_syncer:
+            num_days_to_sync = config.get('num_days_to_sync', 7)
```

Comment:
Based on the discussion I had with Helpshift today, it should be safe to have the default be 180 (or 6 months).

Reply:
I don't think that's necessary unless we're saying that an issue could be updated 6 months after. My assumption is that there wouldn't be updates to data past 30 days so it's not necessary to sync longer than that, but I can change the default to that. Changing it to 180 days would take very long to sync especially if we're planning to run this every day. I haven't timed how long that would take, but it would probably start spilling into busy hours.